### DISCUSSION TITLE
How to add mounted folders in Storage setup

### DISCUSSION THREAD

**Discussion by grunta56** [rern/rAudio#1172] [General]
Using latest release. There is clearly something simple that I am missing and I have read a previous post about use of \ .
When I try to set up a subfolder on my NAS the setup menu does not allow use of /  (I type it and it immediately vanishes) which means I have to mount the whole Drive rather than a subfolder of the drive. I do not want the entire drive mounted and scanned. What am I missing please?
Thankyou

**Comment by Fixu69** [comment]
I have the same problem. I would like to mount //server/share/folder and not only //server/share. Not clear why this is not allowed anymore. :(

Workaround:
nano /etc/fstab
add line:
//server/music/MP3s  /mnt/MPD/NAS/MP3s  cifs  noauto,username=readonlyuser,password=secret,uid=45,gid=45,iocharset=utf8  0  0

**Comment by csch0en** [comment]
> Using latest release. There is clearly something simple that I am missing and I have read a previous post about use of \ . When I try to set up a subfolder on my NAS the setup menu does not allow use of / (I type it and it immediately vanishes) which means I have to mount the whole Drive rather than a subfolder of the drive. I do not want the entire drive mounted and scanned. What am I missing please? Thankyou

Hi. I am using rAudio 20230616 and had no problems mounting diffrent FOLDERS from my Synology CIFS share. The only thing i had to do was using another NAME in "Add Network Share". Result: 2 diffrent mountpoints (folders) aim to the same NAS share.
Looks like this in ->config->system:
/mnt/MPD/NAS/folder1 //synologyNAS/music/folder1
/mnt/MPD/NAS/folder2 //synologyNAS/music/folder2

Maybe this will help.

**Comment by R101G** [comment]
@grunta56 
I think you just need to set up a share in the NAS's equivalent of /etc/samba/smb.conf and specify the path you want. If you then just put the IP of the NAS and the name of the share in rAudio, it should find it.
